controller.py

```diff
@@ -186,16 +235,18 @@ def handle_show_queue_callback(
 
             if len(queue_members) != 0:
                 queue_description = f"{queue_name}:\n" + "".join(map(
-                    lambda member_index: str(member_index[0] + 1) + ". @"
-                    + member_index[1].member_name + "\n",
+                    lambda member_index: member_index[1]
+                    .format_queue_string(member_index[0] + 1),
                     enumerate(queue_members)
                 ))
             else:
                 queue_description = f"{queue_name}:\nОчередь пуста"
 
             self.telegram_message_manager.send_message(
                 callback_query["message"]["chat"]["id"],
-                queue_description
+                queue_description,
+                None,
```

Comment:
We can express this clearer using the name of the argument like this `parse_mode="HTML"`